bin/kano-empty-trash

```diff
@@ -42,17 +43,44 @@ def empty_all_trash():
         # it thinks it must be root if sudo is in effect, but this is not the case here.
         os.system('sudo -H -u#{} SUDO_USER={} /usr/bin/kano-empty-trash '.format(uid, uname))
 
+def clean_old_dumps(dir, age):
+    '''
+    Cleans the core dump files which are set by the kernel.core_pattern
+
+    Args:
+        dir (str): This is where core dump files live.
+        age (int): List files older than age days old.
+    Raises:
+        OSError: Raised if the core dump dir isn there
 
+    TODO:
+        * Get the coredump_dir directly from an environment variable or through
+        the configuration file of kernel.core_pattern 
+    '''
+    if dir:
+        for file in os.listdir(dir):
+            full_path = os.path.join(dir, file)
+            now = datetime.now()
```

Comment:
Do this outside the loop - it doesn't change between iterations so is unnecessary to recalculate.

Reply:
@tombettany Right! I had it outside and only changed to make the code more compact but it introduces not necessary iteration! Thanks for pointing